2_process/src/summarize_targets.R

```diff
@@ -1,5 +1,5 @@
-summarize_targets <- function(ind_file, ...) {
-  ind_tbl <- tar_meta(c(...)) %>%
+summarize_targets <- function(ind_file, vec) {
+  ind_tbl <- tar_meta(vec) %>%
```

Comment:
You may have already noted this from the issue, but `targets` v0.5.0.9000 complains about passing in a vector as "ambiguous". You can get rid of the message by wrapping your file vector argument used in tar_meta() with all_of() in your summarize_targets() function.